sources/PseudoTerminal.m

```diff
@@ -5637,6 +5637,34 @@ - (IBAction)moveTabRight:(id)sender
     [self tabsDidReorder];
 }
 
+- (IBAction)increaseRow:(id)sender {
+    if (![iTermProfilePreferences boolForKey:KEY_DISABLE_WINDOW_RESIZING inProfile:self.currentSession.profile]) {
+        [self.currentSession.screen terminalSetRows:self.currentSession.rows + 1
```

Comment:
Why not call `[self sessionInitiatedResize:self.currentSession width:self.currentSession.columns height:self.currentSession.rows+1]`? It saves several levels of indirection.